src/Observable.php

```diff
@@ -1473,7 +1553,7 @@ public function repeatWhen(callable $notifier)
      * @param SchedulerInterface $scheduler
      * @return AnonymousObservable
      */
-    public function subscribeOn(SchedulerInterface $scheduler)
+    public function subscribeOn(SchedulerInterface $scheduler): AnonymousObservable
```

Comment:
We should get rid of it in v2 - it is easier to add back than it is to get rid of it later.